pkgs/by-name/so/solana-cli/package.nix

```diff
@@ -36,37 +38,26 @@
   ],
 }:
 let
-  version = "1.18.26";
-  hash = "sha256-sJ0Zn5GMi64/S8zqomL/dYRVW8SOQWsP+bpcdatJC0A=";
-  rocksdb = rocksdb_8_3;
+  version = "3.0.0";
```

Comment:
This version is just a testnet release , or did i misunderstand ?
Should we just default to latest stable ? 2.3.8 i think